realm/realm-library/src/objectServer/java/io/realm/mongodb/User.java

```diff
@@ -237,6 +235,15 @@ public Long getMaxAge() {
         return identities;
     }
 
+    /**
+     * Returns the provider type used to log the user
+     *
+     * @return the provider type of the user
+     */
+    public String getProviderType() {
```

Comment:
We should return the `Credentials.IdentityProvider` enum here instead of the underlying String.